### DISCUSSION TITLE
Connection limit in Roles sections

### DISCUSSION THREAD

**Discussion by farzd** [supabase/supabase#1747] [Questions] [answered]
I don't understand this 0/100 metric? Does this mean i can only have 100 concurrent connections to my db?

![image](https://user-images.githubusercontent.com/1423413/119980903-d8f6bf00-bfb4-11eb-9ad7-ff79d83ff992.png)

**Comment by awalias** [comment, marked as answer]
hey @farzd this number on the dashboard just represents the maximum number of *direct* connections per database role, as you can see in the table [we use pgbouncer as a connection pooler](https://supabase.io/blog/2021/04/02/supabase-pgbouncer) which means that any direct connections made on port 6543 will only be kept open for the duration of the transaction itself and any new transactions over the limit will be queued until a slot becomes available meaning that you can successfully serve many more transactions than the number of connection slots available. The pgbouncer config can be managed from `https://app.supabase.io/project/<YOUR-PROJECT-REF>/database/pooling`

also note that if you're using the REST API (or supabase-js etc.) then all these requests will be made via a single connection that our API server (postgrest) keeps open with the db, again meaning that you can handle many simultaneous requests ([upwards of 1200 requests per second](https://supabase.io/beta#performance) from up to the same number of origins)

**Reply by farzd** [reply]
Awesome. Thank you for getting back to me!
